buildservice.proto

```diff
@@ -20,6 +20,7 @@ message PushFileResponse {
 
 message DeleteMessage {
   string uid = 1;
+  string edge = 2;
```

Comment:
Is this really what we want? How about using oneof?

Reply:
@hemarkus We need both variables to disconnect. We need to know the uid of the node and the edge that we want to remove.

Reply:
I see ...